src/main/java/core/basesyntax/service/operations/OperationsStrategyImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+import java.util.Map;
+
+public class OperationsStrategyImpl implements OperationsStrategy {
+    private final Map<Operations, OperationsHandler> operationsHandlerMap;
+
+    public OperationsStrategyImpl(Map<Operations, OperationsHandler> operationsHandlerMap) {
+        this.operationsHandlerMap = operationsHandlerMap;
+    }
+
+    @Override
+    public OperationsHandler get(Operations operation) {
+        return operationsHandlerMap.get(operation);
```

Comment:
It's a good practice to check if the `operationsHandlerMap` contains the key before getting the value to avoid a possible `NullPointerException`. Consider throwing a custom exception if the operation is not supported.